src/BenGorUser/User/Domain/Model/User.php

```diff
@@ -192,9 +189,17 @@ public function id()
 
     /**
      * Accepts the invitation request.
+     *
+     * @throws UserTokenExpiredException when the token is expired
      */
     public function acceptInvitation()
     {
+        if (null === $this->isInvitationTokenExpired()) {
```

Comment:
`isInvitationTokenExpired()` returning `null` is quite confusing. Maybe we need two checks `isInvitationTokenAccepted` and `isInvitationTokenExpired`